ci/run.js

```diff
@@ -0,0 +1,51 @@
+var spawn = require("cross-spawn");
+var os = require("os");
+
+var browser = os.platform() === "win32" ? "" : ":phantomjs";
+
+// The pyramid of doom :( If you have a better idea, (which doesn't
+// involve wait.for / node-gyp / fibers) please create a pull request,
+// but bear in mind that this has to run on windows machines as well...
+execute("examples/angular2", "npm run dev:ci" + browser, function() {
```

Comment:
The `npm run dev:ci` string is copied, so you could pull that into a separate constant variable I suppose, maybe

Reply:
Needs a bit of cleanup :)